hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1725,23 +1627,7 @@ public void restart() throws IOException {
     metrics.setNumBuckets(metadataManager.countRowsInTable(metadataManager
         .getBucketTable()));
```

Comment:
I think this (and the above 3 lines) should be removed too, as we are calling these in the `setMetricsFromDB()` method.

Reply:
Good catch, I'll fix this in the next commit